### PR TITLE
Nathalie&Sofia&Trista-project-survey-vite

### PR DESCRIPTION
**Pull request by trista1987**
## Netlify link
https://pair-project-survey-vite.netlify.app/

## Collaborators
[ohitsnathalie, sofia32057, trista1987]

### CODE REVIEW

File: src/App.jsx [hunk 1]
```diff
@@ -1,3 +1,200 @@
...
+  const optionA = "You heroically wrestled a rogue chocolate fountain.";
+  const optionB = 
+    "You attempted a chocolate chip cookie dough 'quality check' with questionable hygiene.";
+  const optionC =
+    "You tripped face-first into a Halloween candy piñata. 🪅 We've all been there";
+  const optionD =
+    "You have a pet sloth with a mysterious chocolate addiction. Don't judge";
+
+  return (
+    <>
```
Comment (Citronskal) [inline src/App.jsx:41]: It's very smart to use shorthand syntax !

File: src/App.jsx [hunk 2]
```diff
@@ -1,3 +1,200 @@
...
+              aria-label="Your answer">
+              <h2>
+                Your answers:
+              </h2>
+              <h3>
+                  <span>1.</span> When faced with a Halloween bucket full of
+                  candy, you strategically snatch the:
+                </h3>
+              <p>{selectedValue}</p>
+              <h3>
```
Comment (Citronskal) [inline src/App.jsx:63]: was confused by this in the beginning, but then I realized it was for accessbility! Good work!

File: src/App.jsx [hunk 3]
```diff
@@ -1,3 +1,200 @@
...
+                  <option value={optionD}>
+                    You have a pet sloth with a mysterious chocolate addiction.
+                    Don&apos;t judge
+                  </option>
+                </select>
+              </label>
+
+              <label aria-label="Question 3">
+                <h2>
+                  <span>3.</span> Be honest, have you ever tried to sneak candy
```
Comment (Citronskal) [inline src/App.jsx:178]: I am curious about why use span element inside the h2 element if it's not for additional styling? Is it also for accessibility?

**Review by Citronskal — COMMENTED** [review]
Hi Trista, Nathalie and Sofia,

Amazing work as usual! Your survey is fun and creative, codes are clean and structured!
I personally don't have many advice for improvement but just questions :)

**Review by AntonellaMorittu — APPROVED** [review]
Hello and congrats with your Survey project! The app satisfies the requirements and nailed responsiveness and accessibility, it is maybe a bit too simple for being a trio project, but still you delivered :)  👏

**Pull request closed without merging**